website/src/css/_colors.scss

```diff
@@ -0,0 +1,93 @@
+// Colors
+
+@mixin detox-colors-light($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
+  --#{$prefix}-y0: #f7ba29ff;
```

Comment:
Besides, those r,y,b,u,e colors are not inherently dark or light. They are a common part 🤷‍♂️